client/app.jsx

```diff
@@ -65,7 +65,7 @@ class Location extends React.Component {
     return(
       <div>
         <h2>Location</h2>
-        {(this.state.lat !== null && this.state.lng !== null) ?  <Map id={'map'} style={{ width: '100%', height: 300}} lat={this.state.lat} lng={this.state.lng}/> : <div>loading...</div>}
+        {(this.state.lat !== null && this.state.lng !== null) ?  <Map id={'map'} style={{zIndex: -1, position: 'relative', width: '100%', height: 300, borderRadius: '12px'}} lat={this.state.lat} lng={this.state.lng}/> : <div>loading...</div>}
         <About open={this.state.showModal} onClose={this.toggleModal} city={this.state.city} state={this.state.state} country={this.state.country} description={this.state.description} gettingAround={this.state.gettingAround} lat={this.state.lat} lng={this.state.lng}/>
         <h3>{this.state.city}, {this.state.state}, {this.state.country}</h3>
```

Comment:
if you want to I believe you can pass styledComponents as props or even dynamically pass them

Reply:
I noticed that's what you all have been doing. I decided to do that after I get the deployment to work. BTW yours look wonderful!